delete.go

```diff
@@ -2,19 +2,11 @@ package buildah
 
 import (
 	"fmt"
-	"os"
 )
 
 // Delete removes the working container.  The Builder object should not be used
```

Comment:
Should the comment say "The Container object" rather than "The Builder object" here?